### DISCUSSION TITLE
Signature Pad filled %

### DISCUSSION THREAD

**Discussion by KyleEvangelistoRI** [szimek/signature_pad#749] [Ideas]
It would be really cool if there was a function that you could call on the instance of the signature pad that would return what % filled the signature pad is. Like `let percent = signaturePad.getPercentFilled(); `or something. I have a problem where sometimes our users just put a dot and don't actually signed. So if I could see if they filled like 5% at least of the pad that would be cool.

**Comment by UziTech** [comment]
You can do this by counting up the colored pixels in the canvas and dividing them by the total number of pixels. There is no reason this has to be done in signature_pad. There might already be an npm package that does this.

**Reply by KyleEvangelistoRI** [reply]
Good idea, why didn't I think of that lol. Thanks for your time responding
